doc/cephadm/troubleshooting.rst

```diff
@@ -118,23 +118,46 @@ cephadm log file called ``ceph.cephadm.log`` on all monitor hosts (see
 Gathering log files
 -------------------
 
-Use journalctl to gather the log files of all daemons:
+General Procedure for Gathering Log Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. note:: By default cephadm now stores logs in journald. This means
-   that you will no longer find daemon logs in ``/var/log/ceph/``.
+Use ``journalctl`` to gather the log files of all daemons.
 
-To read the log file of one specific daemon, run::
+.. note:: Cephadm stores logs in journald by default. 
+          Daemon logs are not stored in ``/var/log/ceph/``.
+
+Reading the log file of a specific daemon
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To read the log file of one specific daemon, run a command of the following
+form:
+
+.. prompt:: bash #
 
     cephadm logs --name <name-of-daemon>
 
-Note: this only works when run on the same host where the daemon is running. To
-get logs of a daemon running on a different host, give the ``--fsid`` option::
+.. note:: This works only when run on the host where the daemon is running.
+          To get the logs of a daemon running on a different host, use the 
+          ``--fsid`` option.
+
+To read the log file of a specific daemon that is running on a different 
+host than the one whose command line you are currently using, run a
+command of the following form:
 
-    cephadm logs --fsid <fsid> --name <name-of-daemon>
+.. prompt:: bash #
+
+  cephadm logs --fsid <fsid> --name <name-of-daemon>
+
+where ``<fsid>`` corresponds to the cluster ID returned by 
+the command ``ceph status``.
```

Comment:
no.`--fsid` enables you to have multiple clusters running on the **same** host. In case there is more than a single cluster on a host, users need to add the `--fsid` argument. `cephadm logs` doesn't have the ability to go to a different host.